tools/gen-unicode.py

```diff
@@ -371,8 +460,12 @@ def read_case_mappings(unicode_data_file, special_casing_file):
             letter_id = int(line[0], 16)
             condition_list = line[4]
 
-            # Skip supplementary planes, ascii chars, and condition_list
-            if letter_id >= 0x10000 or letter_id < 128 or condition_list:
+            # Skip non-supplementary planes
+            if plane_type == 'supplementary':
```

Comment:
It is strange for me to see strings for types. Could we use enum constants similar to debugger?